modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -982,22 +1025,62 @@ resource page will be rendered.
 component to be rendered when the model matches
 |===
 
+[discrete]
+== `console.storage-provisioner`
+
+Adds a new storage class provisioner as an option during storage class creation.
+
+[cols=",,,",options="header",]
+|===
+|Name |Value Type |Optional |Description
+|`CSI` |`ProvisionerDetails` |yes | Container Storage Interface provisioner type
+
+|`OTHERS`
+|`ProvisionerDetails`
+|yes
+|Other provisioner type
+|===
+
 [discrete]
 == `console.storage-provider`
 
+This extension can be used to contribute a new storage provider to select, when attaching storage and a provider specific component.
```

Comment:
also formatting issue for `console.storage-provider` 
![Screenshot 2023-03-30 at 2 08 47 PM](https://user-images.githubusercontent.com/12692381/228744943-84141a0b-76a8-490f-934d-2c334df95022.png)